package.json

```diff
@@ -154,4 +154,4 @@
         "report-dir": "coverage/cypress/"
     },
     "dependencies": {}
-}
```

Comment:
Did you mean to remove this white space? I think files are supposed to have one blank line at the end?